apps/cowswap-frontend/src/modules/tradeSlippage/state/slippageValueAndTypeAtom.ts

```diff
@@ -22,24 +22,20 @@ const ethFlowSlippageAtom = atomWithStorage<SlippageBpsPerNetwork>(
   mapSupportedNetworks(undefined),
 )
 
-const smartTradeSlippageInnerAtom = atom<number | null>(null)
+export const isSmartSlippageEnabledByWidgetAtom = atom<boolean>(false)
```

Comment:
Since we have https://widget.cow.fi, it will be a bit confusing name.
Suggestion: `isSmartSlippageEnabledByWidgetAtom` -> `shouldUseAutoSlippageAtom`